pages/open_cas_linux/getting_started_open_cas_linux.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Getting Started
+last_updated: April 29, 2019
+toc: false
+permalink: getting_started_open_cas_linux.html
+---
+
+This page describes the basic steps to get started on Open CAS Linux
+
+## About
+Open CAS Linux builds upon the Open CAS Framework by implementing kernel adapters for the Linux operating system.
+The Open CAS Linux project is hosted in this [**GitHub repository**](https://github.com/Open-CAS/open-cas-linux).
+
+## Getting Started
+1. First clone the Open CAS Linux project
+   > ```git clone https://github.com/Open-CAS/open-cas-linux```
+
+2. Change current directory to project folder and update submodules
+   > ```cd open-cas-linux```   
+   > ```git submodule update --init```
+
+3. Configure Open CAS Linux
+    > ```./configure```
+
+4. Compile Open CAS Linux
+   > ```make```
+
+5. Insert kernel modules
```

Comment:
Why not just run 'make install'? This would insert modules as well as install casadm & all related scripts/files :)